lib/task_listener.js

```diff
@@ -341,5 +341,7 @@ export default class TaskListener extends EventEmitter {
         });
       }
     }
+
+    this.runtime.stats.record('taskError');
```

Comment:
Yea, this is to catch things that bubble up and were not caught elsewhere or when getting a device or claim status throws an exception at the task_listener.js level.  We can add something here to log something.